src/main/java/br/ufpe/cin/files/FilesManager.java

```diff
@@ -313,9 +314,7 @@ public static boolean writeContent(String filePath, String content){
 					file.getParentFile().mkdirs();
 					file.createNewFile();
 				}
-				BufferedWriter writer = Files.newBufferedWriter(Paths.get(filePath));
-				writer.write(content);
-				writer.flush();	writer.close();
+				FileUtils.write(file, content);
 			} catch(NullPointerException ne){
```

Comment:
@jvcoutinho hasn't these kinds of issues been resolved before?

Reply:
Well, I haven't detected this specific one, but I don't know if my solutions cover this, maybe. Anyway, they haven't been integrated yet and FileUtils seems better to use.
It also looks like an encoding problem.

Reply:
@jvcoutinho could you then please fix that?